src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -128,7 +128,7 @@ function DiscoverDocumentsComponent({
   const useNewFieldsApi = useMemo(() => !uiSettings.get(SEARCH_FIELDS_FROM_SOURCE), [uiSettings]);
   const hideAnnouncements = useMemo(() => uiSettings.get(HIDE_ANNOUNCEMENTS), [uiSettings]);
   const isLegacy = useMemo(() => uiSettings.get(DOC_TABLE_LEGACY), [uiSettings]);
-  const sampleSize = useMemo(() => uiSettings.get(SAMPLE_SIZE_SETTING), [uiSettings]);
+  const defaultSampleSize = useMemo(() => uiSettings.get(SAMPLE_SIZE_SETTING), [uiSettings]);
```

Comment:
- Is there a max limit? If there isn't, we need to have one set. Users often don't know the effects of loading too many documents.

- What's the default for sample size? One test uses 70, is that it?

Reply:
Hi @TinaHeiligers,

> Is there a max limit? If there isn't, we need to have one set. Users often don't know the effects of loading too many documents.

Yes, currently it's between 10 and 1000. Users can now also press "Load more" for loading next pages ([PR](https://github.com/elastic/kibana/pull/163784)).

> What's the default for sample size? One test uses 70, is that it?

It's 500 by default.